web/libs/editor/src/lib/AudioUltra/Visual/Renderer/Plugins/ProgressRendererPlugin.ts

```diff
@@ -191,10 +195,10 @@ export class ProgressRendererPlugin implements RendererPlugin<ProgressRendererPl
     text.style.marginBottom = "1px";
 
     if (cachingTotal === 0) {
-      let fftCacheCount = 0;
+      let _fftCacheCount = 0;
       if (fftCache) {
         for (const channelMap of fftCache.values()) {
-          fftCacheCount += channelMap.size;
+          _fftCacheCount += channelMap.size;
```

Comment:
@cloudmark This value is unused, can we remove it and what are the impacts if we do?